test/uk/gov/hmrc/helptosave/services/UserCapServiceSpec.scala

```diff
@@ -171,21 +175,21 @@ class UserCapServiceSpec extends TestSupport with ServicesConfig {
       val userCapService = new UserCapServiceImpl(userCapStore, configOverride)
 
       "allow account creation as long as dailyCap is not reached with no record today " in {
-        val userCap = UserCap(yesterday, 9, 100)
+        val userCap = UserCap(yesterday, dailyLimit, totalLimit)
```

Comment:
Should this be the following?
val userCap = UserCap(today, dailyLimit, totalLimit - 1)